physical/raft/raft.go

```diff
@@ -1030,6 +1031,12 @@ func (b *RaftBackend) SetRestoreCallback(restoreCb restoreCallback) {
 	b.fsm.l.Unlock()
 }
 
+func (b *RaftBackend) SetRemovedCallback(cb func()) {
```

Comment:
```suggestion
func (b *RaftBackend) setRemovedCallback(cb func()) {
```

Can we leave this function unexported?

Reply:
We will still be able to access it in `physical/raft/raft_test.go` if it's unexported, unless you mean for future tests outside this PR? Either way no biggie, feel free to resolve the thread 😄

Reply:
True, I was thinking for any future tests